File: 1732-find-the-highest-altitude/1732-find-the-highest-altitude.cs
public class Solution {
    public int LargestAltitude(int[] gain) {
        List<int> list = new List<int>();
        list.Add(0);
        int startedTrip = 0 + (gain[0]);
        list.Add(startedTrip);
        for(int i = 1; i < gain.Length; i++){
            int altitude = startedTrip + (gain[i]);
            startedTrip = altitude;
            list.Add(altitude);
        }
        
        return list.Max();
        
    }
}

File: 1732-find-the-highest-altitude/1732-find-the-highest-altitude.py
class Solution:
    def largestAltitude(self, gain: List[int]) -> int:
        res = [0]
        started_trip = 0 + (gain[0])
        res.append(started_trip)
        for i in range(1, len(gain)):
            alt = started_trip + (gain[i])
            started_trip = alt
            res.append(alt)

        return max(res)
        